snmp/datadog_checks/snmp/data/default_profiles/avocent-acs.yaml

```diff
@@ -0,0 +1,70 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+  - _generic-ucd.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.10418.26.1.* # Avocent Console Server
+  - 1.3.6.1.4.1.10418.26.1.1 # Avocent acs8048
+  - 1.3.6.1.4.1.10418.26.1.2 # Avocent acs8032
+  - 1.3.6.1.4.1.10418.26.1.3 # Avocent acs8016
+  - 1.3.6.1.4.1.10418.26.1.4 # Avocent acs8008
+  - 1.3.6.1.4.1.10418.26.1.5 # Avocent acs808
+  - 1.3.6.1.4.1.10418.26.1.6 # Avocent acs804
+  - 1.3.6.1.4.1.10418.26.1.7 # Avocent acs802
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "avocent"
+
+metrics:
+#  - MIB: ACS8000-MIB
+#    symbol:
+#      name: acsPowerSupplyStatePw1
+#      OID: 1.3.6.1.4.1.10418.26.2.1.8.2.0
+#      enum:
+#        statePowerOn: 1
+#        statePowerOff: 2
+#        powerNotInstaled: 9999
+#      TODO: enum in scalar metric is not supported yet (keep this metric and this
+#        comment in profile until it's fixed)
+#  - MIB: ACS8000-MIB
+#    symbol:
+#      name: acsPowerSupplyStatePw2
+#      OID: 1.3.6.1.4.1.10418.26.2.1.8.3.0
+#      enum:
+#        statePowerOn: 1
+#        statePowerOff: 2
+#        powerNotInstaled: 9999
+#      TODO: enum in scalar metric is not supported yet (keep this metric and this
+#        comment in profile until it's fixed)
+  - MIB: ACS8000-MIB
```

Comment:
What's the meaning of ACS "natural" prefix?

Reply:
Something like Advanced serial console, not sure
Just realised ACS also mean Access Control Server in networking terminology, WDYT of prefixing it by `avocent` ?

Reply:
1.3.6.1.4.1.10418 -> avocent
1.3.6.1.4.1.10418.26 -> acs